back/prisma/schema.prisma

```diff
@@ -394,6 +395,22 @@ model FormGroupement {
   @@index([initialFormId], map: "_FormGroupementInitialFormId")
 }
 
+// Entreprise jouant le rôle d'intermédiaire sur un bordereau
+model IntermediaryFormAssociation {
+  id        String   @id @default(cuid())
+  formId    String?
```

Comment:
Est-ce qu'on ne devrait pas le rendre `NOT NULL` ? A priori c'est obligatoirement lié à un bordereau ?